lib/runctl.js

```diff
@@ -239,3 +239,12 @@ function requestOfTarget(req, rsp, callback) {
     callback(rsp);
   }
 }
+
+function clusterStatus() {
+  var mStatus = master.status();
+  // pst is used by PM, startTime is set by strong-cluster-control
+  mStatus.master.pst = mStatus.master.pst || mStatus.master.startTime;
+  mStatus.appName = agent().config.appName;
```

Comment:
Ah right, that was the case I meant to tackle.. got distracted by the mysterious test failure. I'll fix this up.